core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/DishInOrder.java

```diff
@@ -26,4 +24,78 @@ public class DishInOrder {
     @ManyToOne
     @MapsId("orderId")
     private Order order;
+
```

Comment:
Code smell! There is no reason to implement getters, setters, etc. if you are using lombok. Also, consider not to implement toString for entities. Read [first](https://vladmihalcea.com/the-best-way-to-implement-equals-hashcode-and-tostring-with-jpa-and-hibernate/) [second](https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/) [third](https://vladmihalcea.com/hibernate-facts-equals-and-hashcode/)